interpreter/nodev8/v8.go

```diff
@@ -1924,8 +2002,13 @@ func (d *v8Data) readIntrospectionData(ef *pfelf.File, syms libpf.SymbolFinder)
 	}
 	if vms.Code.InstructionSize != 0 {
 		if vms.Code.SourcePositionTable == 0 {
-			// At least back to V8 8.4
-			vms.Code.SourcePositionTable = vms.Code.InstructionSize - 2*pointerSize
+			if d.version >= v8Ver(12, 4, 0) {
+				// [XXX] how far back?
+				vms.Code.SourcePositionTable = vms.Code.InstructionStart - 3*pointerSize
+			} else {
+				// At least back to V8 8.4
+				vms.Code.SourcePositionTable = vms.Code.InstructionSize - 2*pointerSize
+			}
```

Comment:
Any possibility to detect this from presence of symbol instead of version?

Reply:
Done